spec/takeaway_spec.rb

```diff
@@ -0,0 +1,67 @@
+require 'takeaway'
+
+describe Takeaway do
+
+  let(:items) { double :items }
+  let(:subject) { Takeaway.new }
+  
+  context "View items" do 
```

Comment:
Its really good how you split your main describe block into these context blocks.